incubator/hnc/internal/validators/object.go

```diff
@@ -246,7 +246,19 @@ func (o *Object) hasConflict(inst *unstructured.Unstructured) (bool, []string) {
 	// Get a list of conflicting descendants if there's any.
 	for _, desc := range descs {
 		if o.Forest.Get(desc).HasSourceObject(gvk, nm) {
-			conflicts = append(conflicts, desc)
+			// check if the selectors match
```

Comment:
This is a fairly low-level comment. Can you think of a higher-level description of what you're checking for here?

Reply:
updated the comment to "// If the user have chosen not to propagate the object to this descendant, there shouldn't be any conflict reported here"

Reply:
Perfect